lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -19,10 +19,14 @@
  *
  * @link https://core.trac.wordpress.org/ticket/53494.
  *
+ * @deprecated 6.1 Block supports styles are now stored for enqueuing via the style engine API. See: packages/style-engine/README.md.
+ *
  * @param string $style String containing the CSS styles to be added.
  * @param int    $priority To set the priority for the add_action.
  */
 function gutenberg_enqueue_block_support_styles( $style, $priority = 10 ) {
+	_deprecated_function( __FUNCTION__, '6.1' );
```

Comment:
Should we remove this deprecation notice since it didn't make it into 6.1, and instead add it back in once we're ready to start adding in `6.2` versions of these functions?

Reply:
Oh yeah 🤦 This PR is old. 

Will do, thank you!

Reply:
Looks like there's a wordpress-6.2/script-loader.php file now. I could move it there and keep the notice, and bump it to 6.2